<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_kademarstart</class>
 <widget class="QWidget" name="Form_kademarstart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>kademarstart</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../usr/share/icons/default.kde/32x32/apps/tux.png</normaloff>../../../../../../usr/share/icons/default.kde/32x32/apps/tux.png</iconset>
  </property>
  <widget class="QTabWidget" name="pages">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>641</width>
     <height>397</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_ppal">
    <attribute name="title">
     <string>Principal</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>640</width>
       <height>375</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../usr/share/kademar/utils/kademarcenter/img/bg/bg_principal.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="logo_kademar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../usr/share/kademar/utils/kademarcenter/img/logo.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="l_benvinguts">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Welcome to</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_versio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>83</y>
       <width>400</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Benvinguts al vostre nou sistema operatiu kademar.

Pot començar a utilitzar el sistema o, si vols, pots veure les novetats, configurar i altres temes relacionats.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_autostart">
     <property name="geometry">
      <rect>
       <x>358</x>
       <y>344</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>No tornar a engegar a l'inici</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_suport">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Suport i Manuals</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/suport.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/suport.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_equip">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Equip kademar
(Qui som?)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/equip1.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/equip1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_configurar">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Configurar el Sistema</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/icons/oxygen/48x48/categories/preferences-system.png</normaloff>../../../../../usr/share/icons/oxygen/48x48/categories/preferences-system.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_notes">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Notes de la Versió</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/notes.png</normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/notes.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>l_benvinguts</zorder>
    <zorder>l_versio</zorder>
    <zorder>logo_kademar</zorder>
    <zorder>cb_autostart</zorder>
    <zorder>b_suport</zorder>
    <zorder>b_equip</zorder>
    <zorder>b_configurar</zorder>
    <zorder>b_notes</zorder>
   </widget>
   <widget class="QWidget" name="tab_notes">
    <attribute name="title">
     <string>Notes de la Versió</string>
    </attribute>
    <widget class="QWebView" name="web_notes">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>370</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>file:///usr/share/kademar/utils/kademarcenter/html/notesversio_ca.html</string>
      </url>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_configurar">
    <attribute name="title">
     <string>Configurar Sistema</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>640</width>
       <height>375</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../usr/share/kademar/utils/kademarcenter/img/bg/bg_principal.png</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../usr/share/icons/default.kde/48x48/apps/kde.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Menú K   -&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../usr/share/icons/default.kde/48x48/apps/preferences-system-windows-actions.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Centre de Control</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>481</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>kademar ser personalitzada completament, tan en aparença, com els diferents programes que conté.

La major part de les configuracions visuals es poden fer des del Centre de Control.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="b_kcontrol">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>211</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Engegar Kcontrol ara</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_cadi">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>290</y>
       <width>211</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Engegar CADI ara</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>481</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Les configuracions del sistema, així com instal·lar impressores, configurar internet,  gestionar usuaris, configurar els teclats multimèdia o instal·lar nous programes; es fan des del CADI. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>281</width>
       <height>130</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../usr/share/icons/default.kde/48x48/apps/kde.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Menú K   -&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../usr/share/kademar/utils/cadi/img/cadi.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>CADI</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_suport">
    <attribute name="title">
     <string>Suport i Manuals</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>640</width>
       <height>375</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../usr/share/kademar/utils/kademarcenter/img/bg/bg_principal.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="b_faq">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>FAQ
(Preguntas más frecuentes)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/faq.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/faq.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_foro">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Foro de kademar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/foro.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/foro.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_email">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>eMail</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/email.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/email.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_doc">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Documentación en Linea</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/docs.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/docs.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>451</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>El soporte de kademar principalmente es a través de internet.

Puedes configurar internet, sino está funcionando ya en &quot;Configurar Sistema&quot; 

 Una vez tengas conexión, puedes acceder a los servicios de soporte en linea que ofrecemos.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_equip">
    <attribute name="title">
     <string>Equip kademar</string>
    </attribute>
    <widget class="QWebView" name="web_equip">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>370</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>file:///usr/share/kademar/utils/kademarcenter/html/equip_ca.html</string>
      </url>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="l_banner">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>0</y>
     <width>661</width>
     <height>59</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../usr/share/kademar/utils/kademarcenter/img/banda.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sortir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>458</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/exit.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/exit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_enrera">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>458</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/icons/oxygen/16x16/actions/go-previous.png</normaloff>../../../../../usr/share/icons/oxygen/16x16/actions/go-previous.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_band_cat">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>16</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/band_catala.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/band_catala.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="l_kademar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>kademar GNU/Linux</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_band_esp">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>16</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/band_castella.png</normaloff>../../../../../../usr/share/kademar/utils/kademarcenter/img/band_castella.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
